.changeset/common-candies-kiss.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/create-app': minor
+'example-app-next': minor
+'@backstage/canon': minor
```

Comment:
Best have a separate changeset for canon that explains the changes there instead, the new CSS layout etc.

Reply:
Done